server/config/fusor.yaml

```diff
@@ -0,0 +1,27 @@
+:fusor:
+  :content:
+    :rhev:
+      - :product_name: "Red Hat Enterprise Linux Server"
+        :repository_set_name: "Red Hat Enterprise Linux 6 Server (RPMs)"
+        :basearch: "x86_64"
+        :releasever: "6Server"
+
+      - :product_name: "Red Hat Enterprise Linux Server"
+        :repository_set_name: "Red Hat Enterprise Linux 6 Server - Supplementary (RPMs)"
+        :basearch: "x86_64"
+        :releasever: "6Server"
+
+      - :product_name: "Red Hat Enterprise Virtualization"
+        :repository_set_name: "Red Hat Enterprise Virtualization Hypervisor (RPMs)"
+        :basearch: "x86_64"
+        :releasever: "6Server"
+
+      - :product_name: "Red Hat Enterprise Virtualization"
+        :repository_set_name: "Red Hat Enterprise Virtualization Manager 3.5 (RPMs)"
+        :basearch: "x86_64"
+        :releasever: "6Server"
+
+      - :product_name: "JBoss Enterprise Application Platform"
+        :repository_set_name: "JBoss Enterprise Application Platform 6 (RHEL 6 Server) (RPMs)"
+        :basearch: "x86_64"
+        :releasever: "6Server"
```

Comment:
I'm not a huge fan of this file, but I don't have a nicer way of representing this information besides what you did. My fear is typos could wreak havoc on things or if things get out of sync. For now, this is fine. :+1: